rs/nns/governance/canister/canister.rs

```diff
@@ -745,16 +748,18 @@ fn list_node_provider_rewards() {
 
 #[candid_method(query, rename = "list_node_provider_rewards")]
 fn list_node_provider_rewards_(
-    _req: ListNodeProviderRewardsRequest,
+    req: ListNodeProviderRewardsRequest,
 ) -> ListNodeProviderRewardsResponse {
-    let rewards = governance().list_node_provider_rewards(5);
+    let rewards = governance()
+        .list_node_provider_rewards(
+            MAX_LIST_NODE_PROVIDER_REWARDS_RESULTS,
```

Comment:
Is there an advantage of specifying the limit in `canister.rs`, as opposed to keeping the constant inside `governance.rs` (i.e. the `Governance::list_node_provider_rewards` does not have the `limit` param)? I can only see one advantage of not having to set up a test with 25 entries, and I think we usually want `canister.rs` be just conversion and calling functions on the `Governance` struct.

Reply:
Pretty much just so I can test the limit parameter at the Governance level without setting up 25 entries.  Maybe it's not worth it?